assets/js/lesson_5/hw_10.js

```diff
@@ -0,0 +1,230 @@
+/**
+ * I will give names for variables like taskOne, taskTwo , etc.
+ * and add description to every new task
+ */
+
+// 1.Show numbers from 10 to 20 in one line
+
+let taskOne = "";
+
+for (let i = 10; i <= 20; i++) {
```

Comment:
step has to be 0.5

Reply:
Why? It's not the condition. There's in another homework )

Reply:
Yes, sorry. My bad